### DISCUSSION TITLE
Revamp wp_podsrel table to be based on slugs and not IDs

### DISCUSSION THREAD

**Discussion by sc0ttkclark** [pods-framework/pods#6676] [Ideas]
- [ ] Add new columns to wp_podsrel
- [ ] Run script that fills in slugs for any rows that don't have them set
- [ ] Update code to look at new column slugs for wp_podsrel queries
- [ ] Run script that fills in sister_field with slugs instead of IDs

**Comment by jamesgol** [comment]
Don't forget that ACT might not have slugs :)

**Comment by sc0ttkclark** [comment]
Moving to issue: https://github.com/pods-framework/pods/issues/7179
